tests/rptest/tests/end_to_end.py

```diff
@@ -107,7 +107,10 @@ def debug_mode(self):
         """
         return os.environ.get('BUILD_TYPE', None) == 'debug'
 
-    def start_consumer(self, num_nodes=1, group_id="test_group"):
+    def start_consumer(self,
+                       num_nodes=1,
+                       group_id="test_group",
+                       verify_offsets=True):
```

Comment:
does this mean that all e2e tests now verify offsets by default? should that be called out explicitly in a separate commit?

Reply:
those changes are not required, will revert them